### ISSUE TITLE
Infinitive absolute usage & cognate objects

### ISSUE THREAD

**Issue by IsraelLand** [IAHLT/UD_Hebrew#74]
Hi @amir-zeldes

We've got a bunch of these, used for emphasis or to quote,
**קום תקום** פה וועדה מסדרת
את הדה-לגיטימציה הזאת אתם **עושים חזור ועשה**.
אז אפשר שיהיה "**יסר יסרני** יה", אבל "ולמות לא נתנני"

It seems these are the original cognate objects cases, so we should deprel obl:npmod from the main verb.
What should the _other word_ be in regard to POS and features? 
We've discussed it, and if we do VERB, we have no idea what features to apply, another option is to tag NOUN (Masc, Sing...).

On top of this we have 
יותר מלהתבייש לא נותר לי לעשות דבר, אבל אני **מתבייש את בושת** הממשלה הזאת.

which I assume to be a regular case of cognate object - obl:npmod (nevertheless the "et" marker?)

What do you think? 
Thanks

**Comment by amir-zeldes** [comment]
I agree these are cognate objects, so if we went with `obl:npmod` for those, we should probably stick to it (it would have been nice to maybe add some feature or MISC to allow us to find them later, but it's not crucial). 

For the POS I would stay pretty literal - cognate objects are nouns:

עושים  VERB
חזור  NOUN obl:npmod
ו  CCONJ cc
עשה  NOUN conj

Or are you asking about something else?

**Comment by IsraelLand** [comment]
No, I was just interested if we look at them as nouns, if we assume infinitive absolutes.

I'm still not sure what to do with the את of מתבייש את בושת הממשלה, I assume this is still a cognate object, it's just that it's marked as a regular object - either it's a regular obj or still an obl:npmod with a case deprel (bushat, et).
Thank you!

**Comment by amir-zeldes** [comment]
There are two questions here: is it a cognate object? And do we treat cognate objects differently when they are the sole object?

To answer the first one, it is admittedly not the same morphological form as "בייש התביישתי" (if that is even possible). However from a valency perspective I think it is the same: the verb does not license a regular object, but a noun from the same root is allowed, and the effect is that of a normal cognate object.

As for the other question, here you could get away with `obj` because the verb has no other object, and in fact couldn't have such an object, because the verb is intransitive. But what would you do for "אני מבשל את המנה את בישולו של הטבח האהוב עלי"?

I assume you would want `obl:npmod` in that case, and if so, the question boils down to 'is obl:npmod an escape hatch for double objects, or is it a canonical deprel for cognate objects?'

**Comment by IsraelLand** [comment]
Yes, not only that, but also does the presence of את change anything? When dealing with אני מתבייש את בושת הממשלה הזאת

iirc you've regarded רץ ריצה מהירה as obl:npmod, as a canonical cognate object - a) It seems similar to התביישתי בושה b) it's the sole object, but also c) there's no overt את object marker in רץ ריצה, but in התביישתי בושה, there is.

So I think אני מתבייש את בושת הממשלה הזאת suits b) as sole object, & probably a) as quite a canonical cognate object.
The thing is, for c) there's an overt את marker - it shouldn't necessarily make a difference, but there's a precedent for -

יש לנהג **את האפשרות** של עצירה בטוחה

The driver has **the option**.ACC of safe stop 

יש לנהג **אפשרות** עצירה בטוחה

The driver has a safe stop **option**.SUBJ (i.e. there's a safe stop **option**.SUBJ, for the driver)

Which iirc the first אפשרות is tagged as an object, the latter as an nsubj - following either some sort of realization the את hints at a different structure (I'd argue not, but maybe so), or just a convention because of this peculiar structure יש את.

TL;DR -
So, for התביישתי בושה I'd think to do obl:npmod, as it fits רץ ריצה מהירה (i.e. no double objects required, same structure),
but following the יש את example, it may be that we'd want a similar approach - את marked objects require plain acc. obj.

Thank you

**Comment by amir-zeldes** [comment]
Even with את the questions above remain the same IMO. You can have two objects including a cognate one not really licensed by the verb's argument structure, either with or without et. The cooking example above demonstrates it, and since we cannot use two obj dependents in UD, we need a solution there no matter how much we would like to mark it as `obj` when et occurs. This can also occur with intransitives including motion verbs, for example if I say:

  * רצתי את ריצת הטירוף

It's still a motion verb that normally wouldn't take an object. Mind I think these things can also occur in English, which doesn't have guidelines for cognate objects and would therefore probably mark this as `obj`. But the construction is so much more common in Hebrew that we see cases with two objects, including one like this, quite often. In English I don't think I've run into it so far, and we've been annotating UD for a long time...

BTW I don't think the "yesh" cases have anything to do with this, they are a totally different construction. Synchronically I'm not convinced the one you consider to be `nsubj` is really a subject, it could just be a bare object (note that this has no pronominal equivalent, i.e. I can't say יש לי הוא)

**Comment by IsraelLand** [comment]
For possessives mediated by ל obl, right, you're right this is different, but I don't think Hebrew speakers distinguish the two by design, so whenever ה definiteness is needed/possible, את object marker is used, for both existential and possessive -

יש את מה שהציבור רוצה ויש את מה שהאליטות שיושבות במגדלי השן רוצים. 

יש את תל אביב, ויש את שאר הארץ. 

בכל אחד יש את הניצוץ האלוקי

יש את התקווה משנה לשנה שהמדינה לאט לאט מסתגלת לרעיון 

We, perhaps I, might need a clarification on whether את changes tagging, since it is not a clear cut case that יש את = possessive OBJ, יש = existential NSUBJ.
If את indeed changes things, on a role-taking "signaling" POV, it might do so for התביישתי את הבושה as well.

As for the questions, I agree את or no את does not solve the questions (which are independent of one another since they can both satisfy the obl:npmod position w/ or w/o double "objects" positions taken),
but if you're for רצתי ריצה מהירה to be obl:npmod, I think התביישתי את הבושה is similar enough to be obl:npmod,
unless -
a) you don't see רצתי ריצה מהירה similar to התביישתי את הבושה (or you don't think רצתי ריצה is obl:npmod at all),
b) את changes things similar to יש את - if את is present the speaker somehow "signals" this is a "regular" object (which has bearings on יש את as well). I'm talking strictly linguistically, w/o the technicalities of "no double obj allowed" in mind now.

Thank you

**Comment by amir-zeldes** [comment]
> We, perhaps I, might need a clarification on whether את changes tagging

My own synchronic intuition is that the possessive is never nominative, just sometimes not marked by et. For me "yesh le-dani ha-sefer" is a weird archaic register, but still basically transitive (only stylistic). For the existential case it's more complex - I think the version with no "et" is probably `nsubj`. For the "et" version ("yesh et tel aviv") I'm less sure, and ultimately it's probably just a matter of what would you prefer: for all existentials to have `nsubj`, or for all "et" NPs to be `obj`.

> if you're for רצתי ריצה מהירה to be obl:npmod, I think התביישתי את הבושה is similar enough to be obl:npmod

I see it the same way - I think it should be `obl:npmod` (and these types of NPs are explicitly marked accusative in many languages, incl. in Arabic, but also when they occur in European languages). So basically, Acc is not 1:1 `obj`, which is fine I think.

**Comment by IsraelLand** [comment]
Yeah I agree, but I see both as two sides of the same coin, normativelly we shouldn't use et for either, but we do for both. Admittedly less common for existentials, but it might be due to less "required" use of the definite in existentials ("yesh tel aviv veyesh [et] kol hashar" sounds a bit icky to me).

I'm all for existentials to be nsubj regardless of et; this would require me to do a bit of a re write on the guidelines,
does it make sense to you as a rule that all possessives will be obj, and all existentials be nsubj, even with et attached? meaning -
- לדני יש תפוח.obj, לדני יש את התפוח.obj
- יש תפוחים.nsubj, יש את התפוחים.nsubj שאתה אוהב

I think it's a better representation all around, even though the et-with-nsubj cases.
It seems it would be the same for לי **אין** (את ה) סמכות לכך?

As for התביישתי את הבושה, similarly, the actual obl:npmod representation seems more important than a supposed את 1:1 acc meaning.

Thank you

**Comment by amir-zeldes** [comment]
Yes, I agree. It's a statement about Modern Hebrew grammar for sure, but I think it reflects the synchronic state of the language and the status of "et" (although actually even in Biblical Hebrew, "et" does not mean 1:1 object, there are even some subject cases in the Bible)

**Comment by IsraelLand** [comment]
Great. As this is somewhat of a paradigm change, I think it requires some more thinking, so, would that mean -

אין לי את הסמכות 
אין לי סמכות

are both possessives, thus obj? It's different from what we've (or at least me) been doing thus far.

What about -

הי(ת)ה לי את הסמכות
הי(ת)ה לי סמכות

These all seem intuitively the counterpart to יש, as well as its past form, so they should conform as well,
but this is a big change & they might be somewhat of a different breed (ein being some particle, haya an aux.)
Same goes for vice versa, existentials always being nsubj. What do you think? 
Thank you

**Comment by amir-zeldes** [comment]
> It's different from what we've (or at least me) been doing thus far.

Generally, consistency > smartness - if we can change this in all existing data, I would be on board with it, but we don't want a 50-50 mishmash. If we do make this change, then yes, I would apply it also to the past tense forms.

**Comment by IsraelLand** [comment]
For יש only, I think it's doable enough, it'd make an easier case differentiating the cases - disregarding presence/lack of et. for all the past/negative cases, I don't know, there seems to be more of them, I'll run it by everyone and we'll see.
Thanks a lot

**Comment by amir-zeldes** [comment]
Yes, I think it should be possible to do this automatically based on the presence of obl with ל. If everyone is on board I can change it in the public data as well (for the next UD release, this one is already in data freeze).

**Comment by IsraelLand** [comment]
We've agreed to conform יש את ה (also היה, אין) cases to their indefinite forms.
I've amended the guidelines, we'll correct for past usage sometime in the near future, there seems to be not many.

Thank you very much

**Comment by amir-zeldes** [comment]
OK, so they all take nsubj now, regardless of agreement and "et", possessive or existential, yes? 

If so I can change this in IAHLT HTB as well.

**Comment by IsraelLand** [comment]
Yes, but this seems to be what we've (most of us?) been doing anyway, in regards to both existential and possessive, so we only differentiated between "yesh et" as obj. So not much to correct at all.
This seems to be the way Russian UD works as well with (у меня) есть друг , always an nsubj regardless of cases or "I have" vs. "there is" usage, at least according to my very random sampling. 
Hebrew "yesh" behaves really similar, besides the seemingly Yiddish influence of accusative like et here.

I'm not sure if it should go through Nick before, than public, so I'll ask around, but yeah.

Thank you

**Comment by amir-zeldes** [comment]
OK, keep me posted and like I said I can update the other corpus to match

**Comment by IsraelLand** [comment]
Hi @amir-zeldes

We've just implemented this change, so you can update the public one. The Grew code that @shirawigi composed is this -

rule yesh-nsubj {
        pattern {
                X [lemma=“יש“|”אין“|”היה“];
                Y [form=“את“, Case=“Acc”];
                Z [];
                Z -[case]-> Y;
                e: X -[obj]-> Z;
        }
        commands {
                del_feat Y.Case;
                del_edge e;
                add_edge X -[nsubj]-> Z;
        }
}

So for יש, היה, אין lemmas, with Acc case on את adp, we deleted the case, the deprel, and added a new deprel nsubj.
(Mostly to prevent "false cognates" like יש **את**ם אנשים from triggering and keep the case usage as uniform as possible).

Thanks a lot
Israel

**Comment by amir-zeldes** [comment]
OK, I fixed these in the IAHLTwiki public repo in [this commit](https://github.com/UniversalDependencies/UD_Hebrew-IAHLTwiki/commit/af218badd82c82542e2b901874498bf6cf1f3b7a). There were only 9 cases, and most were false positives with infinitive להיות taking an object, when it should have been a copula, like:

שאיפתו הייתה להיות מדריך ספורט

Where "מדריך" was incorrectly tagged as `obj`. BTW I kept the feature `Case=Acc` on את in this construction, I'm not sure if we want that. Here is a complete tree example of a true positive, notice the FEATS:

```CoNLL-U
1-2	הקפלה	_	_	_	_	_	_	_	_
1	ה	ה	DET	DET	Definite=Def|PronType=Art	2	det	_	_
2	קפלה	קפלה	NOUN	NOUN	Gender=Fem|Number=Sing	3	nsubj:pass	_	_
3	הוקמה	הוקם	VERB	VERB	Gender=Fem|HebBinyan=HUFAL|Number=Sing|Person=3|Tense=Past|Voice=Pass	0	root	_	_
4	או	או	CCONJ	CCONJ	_	5	cc	_	_
5	שופצה	שופץ	VERB	VERB	Gender=Fem|HebBinyan=PUAL|Number=Sing|Person=3|Tense=Past|Voice=Pass	3	conj	_	_
6	על	על	ADP	ADP	_	8	case	_	_
7	ידי	יד	NOUN	NOUN	Definite=Cons|Gender=Fem|Number=Plur	6	fixed	_	_
8	מונומאכוס	מונומאכוס	PROPN	PROPN	_	3	obl	_	_
9-10	במאה	_	_	_	_	_	_	_	_
9	ב	ב	ADP	ADP	Definite=Def|PronType=Art	10	case	_	_
10	מאה	מאה	NUM	NUM	Gender=Fem|Number=Sing	3	obl	_	_
11	ה	ה	DET	DET	Definite=Def|PronType=Art	13	det	_	SpaceAfter=No
12	-	-	PUNCT	PUNCT	_	11	punct	_	SpaceAfter=No
13	11	11	ADJ	ADJ	Gender=Fem|Number=Sing|NumType=Ord	10	amod	_	_
14-15	ויש	_	_	_	_	_	_	_	_
14	ו	ו	CCONJ	CCONJ	_	15	cc	_	_
15	יש	יש	VERB	VERB	_	3	conj	_	_
16-17	בה	_	_	_	_	_	_	_	_
16	ב	ב	ADP	ADP	_	17	case	_	_
17	ה	הוא	PRON	PRON	Gender=Fem|Number=Sing|Person=3|PronType=Prs	15	obl	_	_
18	את	את	ADP	ADP	Case=Acc	20	case	_	_
19-20	השילוב	_	_	_	_	_	_	_	_
19	ה	ה	DET	DET	Definite=Def|PronType=Art	20	det	_	_
20	שילוב	שילוב	NOUN	NOUN	Gender=Masc|Number=Sing	15	nsubj	_	_
21-22	האופייני	_	_	_	_	_	_	_	_
21	ה	ה	DET	DET	Definite=Def|PronType=Art	22	det	_	_
22	אופייני	אופייני	ADJ	ADJ	Gender=Masc|Number=Sing	20	amod	_	_
23	של	של	ADP	ADP	Case=Gen	24	case	_	_
24	לבני	לבנה	NOUN	NOUN	Definite=Cons|Gender=Fem|Number=Plur	20	nmod:poss	_	_
25	חרסים	חרס	NOUN	NOUN	Gender=Masc|Number=Sing	24	compound	_	_
26	דקות	דק	ADJ	ADJ	Gender=Fem|Number=Plur	24	amod	_	_
27-29	וביניהן	_	_	_	_	_	_	_	_
27	ו	ו	CCONJ	CCONJ	_	29	cc	_	_
28	ביני	בין	ADP	ADP	_	29	case	_	_
29	הן	הוא	PRON	PRON	Gender=Fem|Number=Plur|Person=3|PronType=Prs	15	conj	_	_
30	מילוי	מילוי	NOUN	NOUN	Gender=Masc|Number=Sing	29	nsubj	_	SpaceAfter=No
31	.	.	PUNCT	PUNCT	_	3	punct	_	_
```

**Comment by IsraelLand** [comment]
Yes, we'd want to **remove** the case=Acc in these cases. That's what we did with the rest of the data. Though for less than 9...
Thank you

**Comment by amir-zeldes** [comment]
Hm, so what do you think the value should be? Nom? Are we saying that "et" can mark nominative?

**Comment by IsraelLand** [comment]
Nothing really, as it doesn't mark the accusative. It's obviously an accusative artifact, there's no denying its original? accusativeness, I think as we have many frozen leftover case markings (עליהם? ירושלימה), similarly if it's omitted, nothing really is changed in meaning, but awkwardness arises because of breaching of speech norms, not of the grammar itself.

Of special importance perhaps is יש תקווה שמשהו ישתנה vs. יש את התקווה שמשהו ישתנה - the definiteness is almost meaningless, but once it's present, one opts for the את as this is the speech norm - but in itself I see nothing really accusative about the indefinite יש תקווה equivalent to קיימת תקווה, קיימת התקווה.
"פה בסיפור הזה אין ויכוחים בין המשרדים, יש פה את משרד החינוך." (=קיים)
Perhaps this is the German-Yiddish obsession to specific definite articles, sometimes just because (when there's other case markings, irregular cases), even when they aren't needed anymore - met & melted with the Russian variant.

**Comment by amir-zeldes** [comment]
I don't disagree with any of that linguistically, but it seems odd to attach "et" as `case`, but then claim that no case is being assigned. I think we should either keep it Acc and say that Modern Hebrew has an accusatively marked subject in some constructions, or say no, the criterion for Nominative is subjecthood, so it should be Nom. The alternative of no Case feature seems a bit strange to me, but maybe ask around the group - if others think this is the way to go then it's not impossible.

**Comment by IsraelLand** [comment]
I think it's like the obl בכל"מ, isn't it? They're also case without any assigned case feature.
Omer observed before that in such cases, obviously it's not typical for UD to attach a non-obl relation (nsubj) to an obl-like construct (case(tikva, et)) such as here. True, but I mentioned this isn't unheard of in our data, היו בתוכם כאלה שהתלבטו is analyzed as כ אלה - case(ele,ka) - but it's still attached as nsubj, obviously, same goes for כ זה, so it's still possible.
I didn't care much for the Case=Acc at first, but iirc the motivation was, when Shira was composing the new rule - to not confuse, mark (with case=Acc) non-accusative (nsubj) cases as such, therefore inconsistent.

**Comment by amir-zeldes** [comment]
OK, try getting a few more votes from the team and we'll make a final decision

**Comment by IsraelLand** [comment]
Okay, I'll let you know.

**Comment by IsraelLand** [comment]
We've decided to leave these cases with no Case feature.
Thank you
